### DISCUSSION TITLE
Warning:  [CII-SR-013] - Name should not be present

### DISCUSSION THREAD

**Discussion by wh1980** [stephanstapel/ZUGFeRD-csharp#517] [General]
When setting `descriptor.Name = "Rechnung"` the following Warning occurs with this validator:
Warning: [CII-SR-013] - Name should not be present
Validator: [https://validator.invoice-portal.de](url)

When I don't set descriptor.Name then the following code will be generated which will result in the same warning as above:
`<ram:Name />`

In InvoiceDescriptor23CIIWriter line 105 in version 16.1.0 **ram:name** should only be set if descriptor.Name is not null.

**Comment by stephanstapel** [comment]
fixed, thanks
